### DISCUSSION TITLE
Mobile version

### DISCUSSION THREAD

**Discussion by lstrzepek** [lstrzepek/obsidian-yt-transcript#15] [Ideas]
Hi!

Because since version 0.7.1 plugin can work also on mobile, main behaviour needs to be adjusted, because selecting text on mobile is not working same way.
First what comes to my mind is to introduce modal (thanks again @trey-wallis) as a intermediate step. So that link from note will be copied first into modal, then validation will appear if provided text is valid yt link. Then user will be able to adjust link or put different  one and then when action will be confirmed transcription will be fetched.

Or maybe there is better way?

**Comment by decaf-dev** [comment]
It seems like the current behavior on mobile is that when you hold down a YouTube link, you have the open to "Edit link" or "Copy URL". A simple solution would be to add to the window the pops up. Something like "View transcript" and then open the same view that currently exists for desktop. Another idea, in addition to that, would be to add it to the 3 dots context menu.

**Comment by decaf-dev** [comment]
<img src="https://github.com/lstrzepek/obsidian-yt-transcript/assets/40307803/dcec1869-9e82-4634-ab13-9519d199df95" width="200"/>

**Reply by mariomui** [reply]
<img width="813" alt="Screen Shot 2023-06-07 at 8 01 14 PM" src="https://github.com/lstrzepek/obsidian-yt-transcript/assets/29618692/777298a0-d68f-4bdc-be42-f03e8d14ec0d">
I made a little button to open YT transcript automatically. The last link in a bullet point will be automatically scraped and injected into ytranscript. 

This works only if the following observations hold true.
1) Nobody takes notes on TWO or more videos at once.
2) Notes on video are atomic.

I've had it developed for a month now and it works pretty well. The only problem i find is that it doesn't auto stack to a tab.
